tests/models/store.py

```diff
@@ -882,3 +886,29 @@ def test_store_create():
             name="another_store.foo",
             parent=tp.directory,
             translation_project=tp)
+
+
+@pytest.mark.django_db
+def test_store_create_templates(templates):
+    project = ProjectDBFactory(source_language=templates)
```

Comment:
I like that we are trying to avoid mess if we have `Templates` and `English` both and project.source_language is `English`. Though `Templates` can be in any language? I'd prefer to have language set explicitly for `Templates`.